backend/app/assets/stylesheets/spree/backend/components/_currency_selector.scss

```diff
@@ -0,0 +1,39 @@
+.currency-selector-symbol {
+  min-width: 2.5em;
+  text-align: center;
+}
+
+.currency-selector {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  padding-left: .5rem;
+  border: 0;
+  background: transparent;
+
+  -webkit-appearance:none;
+  -moz-appearance:none;
+  appearance:none;
+
+  background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='1024' height='640'><path d='M1017 68L541 626q-11 12-26 12t-26-12L13 68Q-3 49 6 24.5T39 0h952q24 0 33 24.5t-7 43.5z'></path></svg>") 90%/12px 6px no-repeat;
```

Comment:
another option might be to use _::before_ / _::after_ selectors and `content: ` property with the encoded symbol (not entirely sure what the svg is) as a value and position it absolutely if the element has `position: relative`